js/app.js

```diff
@@ -0,0 +1,25 @@
+let dead_zombies = 0;
+let small_zombies = 0;
```

Comment:
1) in JS we don't use "_", so...
smallZombies
madZombies
strongZombies

2) variable name "smallZombies" means an array (like: const zombies = [])
so if want to define a counter use variable like "smallZombiesCount"